Neos.Flow/Classes/Configuration/ConfigurationSource/DefaultConfigurationSource.php

```diff
@@ -0,0 +1,53 @@
+<?php
+namespace Neos\Flow\Configuration\ConfigurationSource;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Configuration\Source\YamlSource;
+use Neos\Flow\Core\ApplicationContext;
+use Neos\Utility\Arrays;
+
+class DefaultConfigurationSource implements ConfigurationSourceInterface
```

Comment:
Maybe `MergeConfigurationSource`? Since that is what it *does* opposed to how it's used (as default)?

Reply:
Good idea with the naming of the class 👍

Reply:
Good point, I renamed the class